### PR TITLE
Readme updated for pdcli oncall : OSD-8119

### PR DESCRIPTION
**Pull request by MitaliBhalla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,38 @@ This will list all the alerts belonging to that incident.
...
+To view the current oncalls as per PagerDuty, use the command:
+
+```
+pdcli oncall
+```
+This will list all the Oncalls for each Escalation policy of **Platform-SRE** team by default.
+
+You can choose to see the Oncalls for all teams on PagerDuty using **All Teams Oncall** on the interactive window.
+
+To view your next Oncall schedule, use **Your Next Oncall Schedule** on the interactive window.
```
Comment (supreeth7) [inline README.md:149]: I feel these lines aren't necessary, you can instead brief the same in the comment column of the oncall view navigation table.

File: README.md [hunk 2]
```diff
@@ -135,6 +135,38 @@ This will list all the alerts belonging to that incident.
...
+|----------------------------------------------------------------|-------------------------------|------------------------------------------------------------------------|
+| All Teams Oncall                                               | `A`                           | Displays Escalations and Oncalls for all teams.                              |
+| Your Next Oncall Schedule                                      | `N`                           | Displays your Oncall schedule.                                        |
+| Go back                                                        | `Esc`                         | Navigate to the previous page.                                         |
+| Quit                                                           | `Q` / `q`                     | Exit the application.                                                  |
+
+
+```
+
+```
```
Comment (supreeth7) [inline README.md:166]: I guess this needs to be deleted.

**Review by supreeth7 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (MitaliBhalla) [reply README.md]: Deleted

File: README.md (hunk 1 above)
Comment (MitaliBhalla) [reply README.md]: Incorporated these

File: README.md [hunk 3]
```diff
@@ -135,6 +135,31 @@ This will list all the alerts belonging to that incident.
...
+pdcli oncall
+```
+### Oncall View Navigation
+
+By default, all the escalations and Oncalls are displayed for team **Platform-SRE** in the main view.
+
+You can choose to view the Oncalls for each Escalation for all teams on PagerDuty using **All Teams Oncall**.
+
+To view your Oncall schedule, use **Your Next Oncall Schedule**.
```
Comment (supreeth7) [inline README.md:151]: I feel this can be omitted because you're explaining the same in the nav table.

Comment (MitaliBhalla) [reply README.md]: omitted

**Review by T0MASD — APPROVED** [review]
/approved
/lgtm
/label tide/merge-method-squash

**Pull request merged**